### DISCUSSION TITLE
how to close install create dev/v3/file

### DISCUSSION THREAD

**Discussion by Soya-xy** [pnpm/pnpm#4808] [Q&A]
After I accidentally upgraded PNPM, when I used PNPM install, the dev/v3/file folder was generated in the current project root directory, which contains all the dependencies packaged.

How do I close it

```
os x:  12.3.1
pnpm  7.1.5
node v18.1.0
```

<img width="1453" alt="image" src="https://user-images.githubusercontent.com/56671014/170641549-bee3f6cd-3931-427f-95ee-376997224c26.png">

**Comment by zkochan** [comment]
It is the global store. You can remove it.

Not sure why it was created inside you project

This is the default location of the store:
Default:
On Windows: ~/AppData/Local/pnpm/store
On macOS: ~/Library/pnpm/store
On Linux: ~/.local/share/pnpm/store

**Reply by Soya-xy** [reply]
yes,  global store also exist，but I don't know why it is also generated in the root directory of the project. If yes, what information do I need to provide? Can you help me analyze the reason
